taskt/Core/Automation/Commands/LoadDictionaryCommand.cs

```diff
@@ -83,15 +84,46 @@ public override void RunCommand(object sender)
 
             engine.AddAppInstance(vInstance, newExcelSession);
 
-
             var excelObject = engine.GetAppInstance(vInstance);
             Microsoft.Office.Interop.Excel.Application excelInstance = (Microsoft.Office.Interop.Excel.Application)excelObject;
 
-            //Query required from workbook using OLEDB
-            DatasetCommands dataSetCommand = new DatasetCommands();
-            DataTable requiredData = dataSetCommand.CreateDataTable(@"Provider=Microsoft.ACE.OLEDB.12.0;Data Source=" + vFilePath + @";Extended Properties=""Excel 12.0;HDR=YES;IMEX=1""", "Select " + v_KeyColumn + "," + v_ValueColumn + " From [" + v_SheetName + "$]");
+            var excelWorkbook = newExcelSession.Workbooks.Open(vFilePath);
+            var excelSheet = excelWorkbook.Sheets[vSheetName];
+ 
+            Microsoft.Office.Interop.Excel.Range last = excelSheet.Cells.SpecialCells(Microsoft.Office.Interop.Excel.XlCellType.xlCellTypeLastCell, Type.Missing);
+            Microsoft.Office.Interop.Excel.Range cellValue = excelSheet.Range["A1", last];
+            
+            List<object> lst = new List<object>();
```

Comment:
Line - 96

Extra Line of code (There is no use of this List)
Please remove this.

Reply:
Done